spec/tests/techniques/technique_sharedFiles.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+file = $params['FILE']
+basename = File.basename(file)
+destination = "/var/rudder/configuration-repository/shared-files"
+
+describe "File Upload" do
+  send_file(file, "/tmp/" + basename)
+end
+
+describe command("chown -r root /tmp/" + basename + "; mv /tmp/" + basename + " " + destination + "/") do
```

Comment:
this doesn't test if chown returned true, you can either split this in 2 tests or use '&&' instead of ';'